dockerfiles/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+mkdir -p $AIRSONIC_DIR/data/transcode
+ln -s /usr/bin/ffmpeg $AIRSONIC_DIR/data/transcode/ffmpeg
+ln -s /usr/bin/lame $AIRSONIC_DIR/data/transcode/lame
+
+chown -R $UID:$GID $AIRSONIC_DIR/data $AIRSONIC_DIR/playlists $AIRSONIC_DIR/podcasts
+
+exec su-exec $UID:$GID tini -- \
+     java -Xmx256m \
+     -Dserver.host=0.0.0.0 \
+     -Dserver.port=$AIRSONIC_PORT \
+     -Dserver.contextPath=/ \
+     -Dlibresonic.home=$AIRSONIC_DIR/data \
+     -Dlibresonic.host=0.0.0.0 \
+     -Dlibresonic.port=$AIRSONIC_PORT \
+     -Dlibresonic.contextPath=/ \
+     -Dlibresonic.defaultMusicFolder=$AIRSONIC_DIR/musics \
+     -Dlibresonic.defaultPodcastFolder=$AIRSONIC_DIR/podcasts \
+     -Dlibresonic.defaultPlaylistFolder=$AIRSONIC_DIR/playlists \
+     -Djava.awt.headless=true \
+     -jar libresonic.war
```

Comment:
Sorry, forget to check master branch changes. I was expected that there is still develop branch that has latest changes.

Fixed in updated commit.